host-controller/src/main/java/org/jboss/as/host/controller/jvm/JvmType.java

```diff
@@ -108,6 +120,20 @@ public Collection<String> getDefaultArguments() {
         return isModularJvm ? DEFAULT_MODULAR_JVM_ARGUMENTS : Collections.EMPTY_LIST;
     }
 
+    public Collection<String> getOptionalDefaultArguments() {
+        if (isModularJvm) {
+            Collection<String> retVal = null;
```

Comment:
Can you add a comment...

```
// The expectation is this method is called once per JvmType instance.
// In theory we could cache the result for reuse, but if we change how
// we use these instances it may not be the case that a cached value
// would be correct over multiple invocations.
```

Reply:
Added. Thanks for feedback.